### PR TITLE
Adds info message after service create and delete operations

### LINKED ISSUE #87: Add a message on stdout after service create
`kn service create` returns without any hint, lets add a message about service creation status.
For eg:
Created <SERVICE_NAME> in <NAMESPACE> namespace.

### PR DESCRIPTION
**Pull request by navidshaikh**
Fixes #87

### CODE REVIEW

File: pkg/kn/commands/service_create.go [hunk 1]
```diff
@@ -73,7 +73,7 @@ func NewServiceCreateCommand(p *KnParams) *cobra.Command {
 			if err != nil {
 				return err
 			}
-
+			fmt.Fprintf(cmd.OutOrStdout(), "Service %s is created in %s namespace.\n", args[0], namespace)
```
Comment (cppforlife) [inline pkg/kn/commands/service_create.go]: i would recommend adding single quotes around names. also consistently putting name after resource type. something like this:

```
Service '%s' was successfully created in namespace '%s'.\n
```

Comment (navidshaikh) [reply pkg/kn/commands/service_create.go]: Ok, updated the PR, also updated the info message for `service delete` too.

File: pkg/kn/commands/service_create.go [hunk 2]
```diff
@@ -16,7 +16,7 @@ package commands
 
 import (
 	"errors"
-
+	"fmt"
```
Comment (cppforlife) [inline pkg/kn/commands/service_create.go]: i think we keeping newline after std imports.

File: pkg/kn/commands/service_create_test.go [hunk 3]
```diff
@@ -79,6 +79,9 @@ func TestServiceCreateImage(t *testing.T) {
 		t.Fatal(err)
 	} else if conf.RevisionTemplate.Spec.Container.Image != "gcr.io/foo/bar:baz" {
 		t.Fatalf("wrong image set: %v", conf.RevisionTemplate.Spec.Container.Image)
+	} else if !strings.Contains(output, "foo") || !strings.Contains(output, "default") || !strings.
```
Comment (cppforlife) [inline pkg/kn/commands/service_create_test.go]: pretty awkward to keep `strings.` on one line. lets end this line at `||`

**Pull request merged**